### DISCUSSION TITLE
Welcome to Python-Bot-Detector Discussions!

### DISCUSSION THREAD

**Discussion by Ferrariic** [Bot-detector/Bot-Detector-Core-Files#1] [General]
Offer suggestions and ideas for the plugin below!

**Comment by ajh6370** [comment]
Where are you looking to take the plugin next? I can try and contribute with my bit of experience with neural networks.

**Reply by Ferrariic** [reply]
That would be great! Pull reqs should be allowed - I'd like to allow for passive names to be input into the server, which are then processed and then evaluated automatically with very little outside input required. The output could be a list of names which meet criteria for suspicious behavior, and perhaps I could try and design a friendly website for players to see some of the stats we've pulled/what we've contributed to Jagex.  Haha - again I'm a medical student not a programmer so I'm doing this from a straight hobbyist perspective. Any help is very much appreciated.
